docs/parameters.rst

```diff
@@ -1,277 +1,179 @@
-OpenSCM models should understand all of the following variables.
-
+They should be of the form ``X to Y Flux`` where the material is flowing from ``X`` into ``Y`` (and hence negative values represent flows from ``Y`` into ``X``):
 
 - ``Land to Air Flux|CO2|Permafrost`` (mass carbon / time) - land to air flux of |CO2| from permafrost
 - ``Land to Air Flux|CH4|Permafrost`` (mass methane / time)
 
 
-Other
-*****
-
-Other variables which should be recognised by OpenSCM adapters are given below.
-
-- ``Surface Temperature`` (K) - surface air temperature i.e. tas
-- ``Ocean Temperature`` (K) - surface ocean temperature i.e. tos
-- ``Ocean Heat Content`` (J)
-- ``Sea Level Rise`` (mm)
-
-
-Regions
--------
-
-Similarly to variables, regions are also provided in a hierarchy separated by the ``|`` character.
-To date, regions which are higher in the hierarchy are the sum of all the regions which are one level below them in the hierarchy (be careful of this when looking at e.g. |CO2| concentration data at a regional level).
-
-All OpenSCM adapaters must understand the following regions:
-
-- ``World``
-- ``World|Northern Hemisphere``
-- ``World|Northern Hemisphere|Ocean``
-- ``World|Northern Hemisphere|Land``
-- ``World|Southern Hemisphere``
-- ``World|Southern Hemisphere|Ocean``
-- ``World|Southern Hemisphere|Land``
-- ``World|Ocean``
-- ``World|Land``
-- ``World|R5ASIA``
-- ``World|R5REF``
-- ``World|R5MAF``
-- ``World|R5OECD``
-- ``World|R5LAM``
-- ``World|R5.2ASIA``
-- ``World|R5.2REF``
-- ``World|R5.2MAF``
-- ``World|R5.2OECD``
-- ``World|R5.2LAM``
-- ``World|Bunkers``
-
-
-Configuration
--------------
-
-Each model will have its own set of configuration parameters and conventions.
-In OpenSCM we allow the user to pass these to and from the model via the adapter, following the model's own internal conventions for naming.
-However, we also insist that models understand the following configuration options.
-
-- ``ecs`` (K) - equilibrium climate sensitivity
-- ``tcr`` (K) - transient climate response
-- ``f2xco2`` (W/m\*\*2) - radiative forcing due to a doubling of atmospheric |CO2| concentrations from pre-industrial level
+Standard regions
+----------------
+
+Similarly to variables, regions are also given in a hierarchy.
+Regions which are higher in the hierarchy are the sum of all the regions which are one level below them in the hierarchy (be careful of this when looking at e.g. |CO2| concentration data at a regional level).
+
+.. csv-table:: Gases
+    :header: "Name 0", "Name 1", "Name 2, "Description"
+
+    ``World``
+    ``World``, ``Northern Hemisphere``
+    ``World``, ``Northern Hemisphere``, ``Ocean``
+    ``World``, ``Northern Hemisphere``, ``Land``
+    ``World``, ``Southern Hemisphere``
+    ``World``, ``Southern Hemisphere``, ``Ocean``
+    ``World``, ``Southern Hemisphere``, ``Land``
+    ``World``, ``Ocean``
+    ``World``, ``Land``
+    ``World``, ``R5ASIA``
```

Comment:
can you add a TODO for me here to dig out the links of where all these come from?

Reply:
@swillner ping

Reply:
Sorry ;) Done